deno.jsonc

```diff
@@ -9,5 +9,10 @@
   },
   "exclude": [
     ".coverage/"
-  ]
+  ],
+  // NOTE: Import maps should only be used from test modules.
+  "imports": {
+    "@denops-private/": "./denops/@denops-private/",
+    "@denops-testutil/": "./tests/denops/testutil/"
```

Comment:
Using `@` here is a bit confusing while those never be the denoland/jsr module. Let's use `/` instead.

- `/denops-private/`
- `/denops-testutil/`

Or `denops:` prefix so that developers can easily distinguish like

- `denops:private/`
- `denops:testutil/`

---

- `/`: Usually used for source root prefix
- `@`: JSR use it for namespace prefix
- `$`: Deno Fresh use it for denoland module prefix

Reply:
`denops:` is can not use, it will be recognized as a schema.

```
error: Unsupported scheme "denops" for module "denops:testutil/mock.ts". Supported schemes: [
    "data",
    "blob",
    "file",
    "http",
    "https",
]
```